setup.py

```diff
@@ -15,16 +15,22 @@ def finalize_options(self):
         pass
 
     def run(self):
+        import django
+        from django.core.management import call_command
         from django.conf import settings
+
         settings.configure(
-            DATABASES={'default': {'NAME': ':memory:', 'ENGINE': 'django.db.backends.sqlite3'}},
-            INSTALLED_APPS=('compressedlistfield', 'django.contrib.contenttypes')
+            DATABASES={'default': {
+                'NAME': ':memory:',
+                'ENGINE': 'django.db.backends.sqlite3'
+            }},
+            INSTALLED_APPS=(
+                'django.contrib.contenttypes',
+                'compressedlistfield.test_app.apps.TestConfig',
+            )
```

Comment:
Formatting aside, this replaces `compressedlistfield` (which doesn't need to be an app) with `compressedlistfield.test_app.apps.TestConfig` which makes `compressedlistfield.test_app` an app.